index.js

```diff
@@ -17,3 +17,347 @@ mobileList.forEach((link) => {
     mobileMenu.style.display = 'none';
   });
 });
+
+// popup_window
+
+// popup windows
+
+const ProjectContainer = document.querySelector('.mobile-card');
+
+const mobileProject = [
+  {
+    name: 'Tonic',
+    company: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: 'canopies.png',
+    technologies: ['HTML', 'CSS', 'JAVASCRIPT'],
+    linkToLive: '',
+    linkToSource: '',
+  },
+  {
+    name: 'Multi-Post Stories',
+    company: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: 'available.png',
+    technologies: ['HTML', 'CSS', 'JAVASCRIPT'],
+    linkToLive: '',
+    linkToSource: '',
+  },
+  {
+    id: '3',
+    name: 'Tonic',
+    company: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: 'gym.png',
+    technologies: ['HTML', 'CSS', 'JAVASCRIPT'],
+    linkToLive: '',
+    linkToSource: '',
+  },
+  {
+    name: 'Multi-Post Stories',
+    company: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: 'mobile-art.png',
+    // technologies: ["HTML", "CSS", "JAVASCRIPT"],
+    linkToLive: '',
+    linkToSource: '',
+  },
+];
+
+const allLists = () => {
+  ProjectContainer.innerHTML = '';
```

Comment:
- [ ] Your form validation code does not run, hence the form is submitted even when an email has a capital letter in it. The issue begins from line 25 and links to line 80. if you look at the browser console an `Uncaught TypeError: Cannot set properties of null (setting 'innerHTML')` appears. Please see  a detail explanation of the things I tried to find out about this issue. 

   - On line 25 - you are trying to grab the project container -> `const ProjectContainer = document.querySelector('.mobile-card');`. But the class name you used is not on the HTML page so the `ProjectContainer ` variable returns `null` because the element was not found on the page. The correct class name should be `.mobile-cards`- notice the `s`.  Because of this, on line 80 this code - `ProjectContainer.innerHTML = '';` produces an error message - `Uncaught TypeError: Cannot set properties of null (setting 'innerHTML')`. You will be wondering how this affects the form validation, it does affect it because all the code after line 80 fails to execute. 

On line 219 - wrong class name - desk-card -> correct class name - desktop-cards
On line 292 - const works = document.querySelector('.modal-section'); - not found on the page at page load. **_I see a lot of issues with your code arrangement. I advise you to take off time to fix all these issues._**